user/templates/templates.md

```diff
@@ -132,15 +129,15 @@ After installing a fresh template, we recommend performing the following steps:
 
 3. If desired, [uninstall the old template](#uninstalling).
 
-## Updating
+## Updating Templates
 
 Please see [How to Update](/doc/how-to-update/).
 
 ## Installing Software
 
 Please see [How to Install Software](/doc/how-to-install-software).
 
-## Uninstalling
+## Uninstalling Template
 
```

Comment:
You should make it clear that you cannot remove a template that is used by an app qube, (including e.g. mgmt-disp-vm). There you have to switch the template before attempting removal